sdk/src/androidTest/java/com/mapbox/maps/LifecycleTest.kt

```diff
@@ -0,0 +1,53 @@
+package com.mapbox.maps
+
+import androidx.test.annotation.UiThreadTest
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import com.mapbox.maps.plugin.PLUGIN_LIFE_CYCLE_CLASS_NAME
+import junit.framework.TestCase.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class LifecycleTest {
+
+  @get:Rule
+  val rule = ActivityScenarioRule(EmptyActivity::class.java)
+
+  private lateinit var mapView: MapView
+  private lateinit var mapboxMap: MapboxMap
+  private lateinit var countDownLatch: CountDownLatch
+
+  @Before
+  @UiThreadTest
+  fun setUp() {
+    rule.scenario.onActivity {
+      mapView = MapView(it, MapInitOptions(it, plugins = listOf(PLUGIN_LIFE_CYCLE_CLASS_NAME)))
+      mapboxMap = mapView.getMapboxMap()
+      it.frameLayout.addView(mapView)
+    }
+  }
+
+  @Test
+  fun testOnStart() {
```

Comment:
Ironically I don't see `onStart` being called in this test 😄 
Perhaps we do want to rename it and add another test with calling `onStart`, wdyt?

Reply:
I actually meant another test calling `onStart` explicitly (here it's called automatically because of lifecycle plugin). to verify that we have same behaviour.

Reply:
With Lifecycle plugin, onStart is called automatically, so the default style could be load. This test case is used to verify this process.